lib/commands/exam.js

```diff
@@ -285,6 +299,9 @@ module.exports = TestCommand.extend({
         commands.get('writeExecutionFile')
       );
     };
+    events['testem:module-done-detail'] = function(details) {
+      testemEvents.recordModuleRunDetail(details.name, details.total, details.runtime);
```

Comment:
what if we simplified this to 
```
// notice I also renamed recordModuleRunDetail to recordModuleMetadata
testemEvents.recordModuleMetadata(   
  {
    details.name,
    details.total,
    details.runtime
  }
);
```

Then everywhere we can simplify the params to:
```
recordModuleMetadata(metadata)
addMetadataToModuleMetadata(metadata)
```